nbhttp/websocket/upgrader.go

```diff
@@ -868,9 +869,9 @@ func nextTokenOrQuoted(s string) (value string, rest string) {
 	return "", ""
 }
 
-func (u *connState) readAll(r io.Reader, size int) ([]byte, error) {
+func (wr *wsReader) readAll(r io.Reader, size int) ([]byte, error) {
 	const maxAppendSize = 1024 * 1024 * 4
```

Comment:
could this result in some sort of truncation and if so, should it be configurable?

Reply:
sorry but I don't get the point for `configurable`, what should we config for?